### PR TITLE
Main/feature section/images and content

### PR DESCRIPTION
**Pull request by jennymakki**
**Description:**
I have added images and styled the feature-section. It is responsive and stacks vertically on screens less than 800px. 

Issue #24 

**Type of change:**
- Rewrote the HTML section to change the divs in order to make them usable in the grid. 
- Added two images that are used in the grid. 
- Rewrote the _feature file to make the right object fit into the right grid. 
- Changed syntax to @use 'file' as *.
- Rearranged some padding in the footer (which I worked with last) 

**How Has This Been Tested or should be tested?**
It has been testet on various screens.

**Additional information:**
Screenshot 2024-12-21 at 20 13 48

### CODE REVIEW

**Review by alfredbrannare — APPROVED** [review]
Nice job!

Good solution with the HTML!

**Comment by jennymakki** [comment]
@alfredbrannare Thank you! I had to change in the file _main_movies from ”import” to ”use” because it seemed like I got an error code for the ”imports” when I ran the dev.

File: Grupparbete-biouppgift
Comment (Paajt) [inline Grupparbete-biouppgift:1]: We should probably delete this folder for good 🤔

**Review by Paajt — APPROVED** [review]
Looks good! Good job! 🎆

**Pull request merged**